tools/mage/dev.go

```diff
@@ -69,7 +69,7 @@ func (Dev) Misspell() error {
 var (
 	devDatabases          = []string{"cockroach", "redis"}
 	devDataDir            = ".env/data"
-	devDatabaseName       = "ttn_lorawan"
+	devDatabaseName       = "ttn_lorawan_dev"
```

Comment:
cc: @rvolosatovs

Reply:
This is actually not related to this PR, so, I'll remove it. Still, this should be the database name, since the default value for the DB URI is \`postgresql://root@localhost:26257/ttn\_lorawan\_dev?sslmode=disable\`